src/js/angular/core/directives/rdfresourcesearch/rdf-resource-search.directive.js

```diff
@@ -69,7 +73,7 @@ function rdfResourceSearchDirective($rootScope, $timeout,
       }
 
       $scope.onKeyDown = function(event) {
-        if (event.keyCode === 27) {
+        if ($location.url() !== '/' && event.keyCode === 27) {
```

Comment:
If you remove it from the DOM on the home page (as I think you should), this will become unnecessary.

Reply:
yup